src/main/java/org/sonarlint/intellij/util/SonarLintUtils.java

```diff
@@ -231,11 +234,30 @@ public static void configureProxy(String host, ServerConfiguration.Builder build
       builder.proxy(proxy);
 
       if (httpConfigurable.PROXY_AUTHENTICATION) {
-        builder.proxyCredentials(httpConfigurable.PROXY_LOGIN, httpConfigurable.getPlainProxyPassword());
+        // Different ways to fetch login based on runtime version (SLI-95)
+        try {
+          Object proxyLogin = tryGetProxyLogin(httpConfigurable);
+          if(proxyLogin != null) {
+            builder.proxyCredentials(proxyLogin.toString(), httpConfigurable.getPlainProxyPassword());
+          }
+        } catch (Exception ex) {
+          LOG.warn("Could not fetch value for proxy login", ex);
+        }
       }
     }
   }
 
+  private static Object tryGetProxyLogin(HttpConfigurable httpConfigurable) throws Exception {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00112)